x-pack/plugins/snapshot_restore/public/application/components/restore_snapshot_form/steps/step_logistics/system_indices_overwritten_callout.tsx

```diff
@@ -9,15 +9,21 @@ import { i18n } from '@kbn/i18n';
 import React, { FunctionComponent } from 'react';
 import { EuiCallOut } from '@elastic/eui';
 
-export const SystemIndicesOverwrittenCallOut: FunctionComponent = () => {
+export const SystemIndicesOverwrittenCallOut: FunctionComponent<{
+  featureStates: string[] | undefined;
+}> = ({ featureStates }) => {
   return (
     <EuiCallOut
       data-test-subj="systemIndicesInfoCallOut"
       title={i18n.translate(
         'xpack.snapshotRestore.restoreForm.stepLogistics.systemIndicesCallOut.title',
         {
           defaultMessage:
-            'When this snapshot is restored, system indices will be overwritten with data from the snapshot.',
+            'When this snapshot is restored, system indices {featuresCount, plural, =0 {} other {from {features}}} will be overwritten with data from the snapshot.',
```

Comment:
I'm not too sure about these changes, It seems to me that its quite obvious which ones will be included since they are listed in the featureStates dropdown right next to this callout